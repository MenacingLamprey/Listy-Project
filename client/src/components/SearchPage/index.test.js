import { screen, render } from "@testing-library/react";
import { SearchPage } from './index';

describe('Search Page', () => {
  test('Should render the search page', () => {
    const { container } = render(<SearchPage />);
    expect(container).toMatchSnapshot();
  })

  test('Should not render children while loading', () => {
    render(<SearchPage />);
    screen.getByText(/Loading/);
  })

  test('Should render a greeting if authenticated', () => {
    render(<SearchPage isLoading={false} />);
    screen.getByText(/Hello/);
  })
})