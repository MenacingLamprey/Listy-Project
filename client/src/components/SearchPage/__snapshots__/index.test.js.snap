// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Page Should render the search page 1`] = `
<div>
  <div
    class="container"
  />
</div>
`;
